src/core/logical_layers/renderers/stylesConfigs/mcda/calculations/index.ts

```diff
@@ -58,17 +69,45 @@ class Calculations<T> implements IsomorphCalculations<T> {
       /* square_root: (sqrt(x) - sqrt(min)) / (sqrt(max) - sqrt(min)) */
       case 'square_root':
         return {
-          tX: this.math.sqrt(x),
-          tMin: this.math.sqrt(min),
-          tMax: this.math.sqrt(max),
+          tX: this.math.mult(this.math.sign(x), this.math.sqrt(this.math.abs(x))),
```

Comment:
Can you please update comment of this transformation, it got me confused

Reply:
Done! Good point